.github/CODEOWNERS

```diff
@@ -1 +1,3 @@
 * @vivienlacourba
```

Comment:
@jennyliang220 as discussed you can remove me as one of the code owners of this project.

```suggestion
```